Dockerfile

```diff
@@ -1,24 +1,28 @@
-FROM node:14.18.0-alpine as build
+FROM node:16-alpine as build
```

Comment:
we should use a fixed version of node, in order to avoid unexpected behaviours

Reply:
Using node:16.16.0